packages/webapp/__tests__/SquadTokenPage.tsx

```diff
@@ -256,7 +256,7 @@ describe('invalid token', () => {
     const owner = generateTestOwner();
     const member = generateTestMember(1);
     member.user.id = defaultUser.id;
-    owner.source.members.edges.push({ node: member });
+    owner.source.currentMember = member;
```

Comment:
There should now be a case if the user is blocked we should not redirect. This test is now inaccurate since the member used here is not blocked and did not redirect.